src/main/java/horseshoe/TemplateLoader.java

```diff
@@ -326,21 +327,51 @@ private Template loadPartial(final String name, final Stack<Loader> loaders) thr
 	 * @throws ReflectiveOperationException if an error occurs while dynamically creating and loading an expression
 	 */
 	private TemplateRenderer loadPartial(final Stack<Loader> loaders, final TemplateLoadState state, final String tag) throws LoadException, ReflectiveOperationException {
-		final Matcher matcher;
-		String indentation = null;
+		final boolean doubleArrow = tag.length() > 1 && tag.charAt(1) == '>'; // ">>" uses indentation on first line, no trailing new line
+		final String indentation;
 
-		if (tag.length() > 1 && tag.charAt(1) == '>') { // ">>" uses indentation on first line, no trailing new line
-			matcher = LOAD_PARTIAL_PATTERN.matcher(tag).region(2, tag.length());
+		if (doubleArrow) {
 			state.setStandaloneStatus(TemplateLoadState.TAG_CHECK_TAIL_STANDALONE);
+			indentation = null;
 		} else {
-			matcher = LOAD_PARTIAL_PATTERN.matcher(tag).region(1, tag.length());
 			indentation = state.removeStandaloneTagHead();
 
 			// 1) Standalone tag uses indentation for each line, no trailing new line
 			// 2) Non-standalone tag uses indentation on first line, with trailing new line
 			state.setStandaloneStatus(indentation == null ? TemplateLoadState.TAG_CANT_BE_STANDALONE : TemplateLoadState.TAG_CHECK_TAIL_STANDALONE);
 		}
 
+		final Matcher expressionMatch = PARTIAL_EXPRESSION.matcher(tag);
```

Comment:
I think I would prefer to roll this into the existing pattern. This would also open up the possibility of passing arguments to the template (`{{> ('a' + 'b')(., ..) }}`).

Reply:
Done